pkg/trace/info/stats.go

```diff
@@ -244,6 +339,7 @@ func (s *Stats) isEmpty() bool {
 
 // String returns a string representation of the Stats struct
 func (s *Stats) String() string {
+	// TODO: just marshal this whole thing to JSON
```

Comment:
Why? This is used for outputting stats.

Reply:
Just a random thought while writing, not relevant anymore, removed